app/src/components/dashboard/services/CardService.js

```diff
@@ -2,7 +2,7 @@
 import InfoCardService from "@/components/dashboard/services/InfoCardService";
 import { useState } from "react";
 import Modal from "@/components/dashboard/services/Modal";
-import Image from "next/image";
+import Image from "next/legacy/image";
```

Comment:
why legacy?

Reply:
you're right, the most recent version is next/image
correction done